docs/getting-started/documentation/api_wrapper.md

```diff
@@ -0,0 +1,34 @@
+Manage Your EDC Connector Via Our EDC API Wrapper Java Client Library
```

Comment:
"Manging the EDC Connector via the API Wrapper Java Client Library"

Diese Dokumentation ist open source und soll deswegen neutraler geschrieben werden, weniger "our" und "your".

Generell sollte alles umformuliert werden was irgendwie CaaS referenziert, aber nicht EE-spezifisch ist. Falls es genau um die Unterschiede zwischen CE und EE geht kann man das ja erwähnen.

Ebenfalls soll nichts von uns mehr "EDC XXX" heißen sondern enweder "sovity EDC XXX" oder "XXX".